functional/FunctionalTest.java

```diff
@@ -2370,6 +2705,9 @@ public static void runTests() throws Exception {
     copyObject_test7();
     copyObject_test8();
     copyObject_test9();
+    copyObject_test10();
+    copyObject_test11();
+    copyObject_test12();
```

Comment:
Encryption related tests should only be triggered if `tls` is set up. Similar to how its done here https://github.com/minio/minio-go/blob/master/functional_tests.go#L7654

Reply:
@nitisht @aarushiarya SSE-S3 is also allowed for plain HTTP (no TLS)
SSE-C and SSE-KMS require TLS and SSE-KMS also signatureV4